### ISSUE TITLE
Request textDocument/foldingRange failed

### ISSUE THREAD

**Issue by souporserious** [mdx-js/vscode-mdx#299]
### Initial checklist

- [X] I read the [support docs](https://mdxjs.com/community/support/)
- [X] I read the [contributing guide](https://mdxjs.com/community/contribute/)
- [X] I agree to follow the [code of conduct](https://github.com/mdx-js/.github/blob/HEAD/code-of-conduct.md)
- [X] I searched [issues](https://github.com/search?q=user%3Amdx-js&type=Issues) and couldn’t find anything (or linked relevant results below)

### Affected packages and versions

1.2.1

### Steps to reproduce

- Load an MDX file that imports another MDX file in VS Code
- Follow the symbol of the imported file
- When navigating to the imported file the error should occur

### Expected behavior

No error is thrown when navigating symbols.

### Actual behavior

<img width="467" alt="image" src="https://user-images.githubusercontent.com/2762082/219266896-155681cc-1d71-489b-9e46-66bb98c284cd.png">

Go to output results in the following:

```
[Error - 8:18:22 PM] Request textDocument/foldingRange failed.
  Message: Request textDocument/foldingRange failed with message: Cannot read properties of undefined (reading 'textSpan')
  Code: -32603 
```

### Runtime

Node v17

### Package manager

pnpm

### OS

macOS

**Comment by remcohaszing** [comment]
This is a duplicate of #289. This has been fixed in #290, but not yet released. I intend to create a new release today or tomorrow.

**Comment by souporserious** [comment]
Oops, sorry for the duplicate! I totally missed that searching through the issues 🤦

**Comment by remcohaszing** [comment]
This has just been released in [v1.2.2](https://github.com/mdx-js/vscode-mdx/releases/tag/v1.2.2) :tada:
